src/components/cards/RelationshipsCard.jsx

```diff
@@ -1,106 +1,510 @@
+            />
+          </div>,
+          <div
+            id="roles2-div"
+            style={{ width: 'fit-content', minWidth: 470 }}
+          >
+            <Autocomplete
+              id="roles2"
+              clearOnEscape
+              style={{ marginTop: 12 }}
+              value={currentRole1}
+              options={currentRoles}
+              renderOption={option => (
+                <Text value={option.guid}>{option.label}</Text>
+              )}
+              onChange={(_, newValue) => onChangeRole1(newValue)}
+              getOptionLabel={option => get(option, 'label', '')}
+              getOptionSelected={(option, val) =>
+                option.guid ? option.guid === val : false
+              }
+              renderInput={params => {
+                return (
+                  <TextField
+                    {...params}
+                    style={{ width: 280 }}
+                    variant="standard"
+                    label={intl.formatMessage(
+                      { id: 'SELECT_RELATIONSHIP_ROLE_2' },
+                      {
+                        ind2: selectedIndividualFirstName,
+                      },
+                    )}
+                  />
+                );
+              }}
+            />
+          </div>,
+        ]}
+      </DialogContent>
+      <DialogActions
+        style={{
+          padding: '0px 24px 24px 24px',
+          display: 'flex',
+          alignItems: 'flex-end',
+          flexDirection: 'column',
+        }}
+      >
+        {postRelationshipError && (
+          <CustomAlert
+            style={{ margin: '20px 0', width: '100%' }}
+            severity="error"
+            titleId="SERVER_ERROR"
+          >
+            <Text key={postRelationshipError} variant="body2">
+              {postRelationshipError}
+            </Text>
+          </CustomAlert>
+        )}
+        <div>
+          <Button
+            display="basic"
+            onClick={onCloseDialog}
+            id="CANCEL"
+          />
+          <Button
+            display="primary"
+            onClick={() => onSubmit()}
+            loading={
+              loading ||
+              loadingRelationships ||
+              postRelationshipLoading
+            }
+            disabled={!allValid}
+            id="ADD_RELATIONSHIP"
+          />
+        </div>
+      </DialogActions>
+    </StandardDialog>,
     <Card
       title={title}
       titleId={titleId}
       maxHeight={600}
       renderActions={
         <div>
-          <IconButton
+          {/* <IconButton
             style={{ color: theme.palette.primary.main }}
             aria-label="View chart"
           >
             <ViewChart />
-          </IconButton>
-          <IconButton aria-label="View list">
-            <ViewList />
-          </IconButton>
+          </IconButton> */}
+          <IconButton
+            aria-label="View list"
+            style={{ color: theme.palette.primary.main }}
+          />
```

Comment:
we should actually just comment out the entire renderActions prop for the time being.

Reply:
Done